docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,79 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer that installs EVE-OS on the
```

Comment:
I think you should start by stating the purpose of the verification - that this is useful and needed when trying EVE-OS on a new model of hardware to ...

Otherwise a reader might think they should run the verification image other times e.g., before every install.